app/scripts/components/data-catalog/index.tsx

```diff
@@ -223,31 +148,31 @@ function DataCatalog({ datasets }: DataCatalogProps) {
     onAction(Actions.TAXONOMY_MULTISELECT, { key: item.taxonomy, value: item.id });
   }, [setAllSelectedFilters, allSelectedFilters, onAction]);
 
-  const handleClearTag = React.useCallback((item: OptionItem) => {
+  const handleClearTag = useCallback((item: OptionItem) => {
     setAllSelectedFilters(allSelectedFilters.filter((selected) => selected !== item));
     setClearedTagItem(item);
 
   }, [allSelectedFilters]);
 
-  const handleClearTags = React.useCallback(() => {
+  const handleClearTags = useCallback(() => {
     setAllSelectedFilters([]);
   }, [setAllSelectedFilters]);
 
-  React.useEffect(() => {
+  useEffect(() => {
     if (clearedTagItem && (allSelectedFilters.length == prevSelectedFilters.length-1)) {
       onAction(Actions.TAXONOMY_MULTISELECT, { key: clearedTagItem.taxonomy, value: clearedTagItem.id}); 
       setClearedTagItem(undefined);
     }
   }, [allSelectedFilters, clearedTagItem]);
```

Comment:
I noticed warnings about `useEffect` has missing dependencies throughout this component. Do you mind putting comments on why those dependencies are excluded? It would help me navigate the code better.

Reply:
Sure. We have missing dependencies sprinkled throughout the existing codebase already. We can make start making notes going forward why some arn't included